freeplane/src/main/java/org/freeplane/view/swing/map/MapViewPane.java

```diff
@@ -292,14 +643,97 @@ public void componentResized(ComponentEvent e) {
         });
     }
 
+    private MapOverviewAttachPoint getMapOverviewAttachPoint() {
```

Comment:
Could a double click on the anchor be used to reset the position?

Reply:
Will be available in next commits.